internal/scaffold/role.go

```diff
@@ -148,13 +148,9 @@ func updateRoleFile(role interface{}, roleFilePath string) error {
 	if err != nil {
 		return fmt.Errorf("failed to marshal role(%+v): %v", role, err)
 	}
-	m := &map[string]interface{}{}
```

Comment:
Is there a reason the role was marshalled into JSON in the first place? We can probably change this block to `yaml.Marshal`.

Reply:
Didn't see the comment about a follow-up.